nixos/modules/misc/ids.nix

```diff
@@ -448,7 +448,7 @@ in
       #tcpcryptd = 93; # unused
       firebird = 95;
       keys = 96;
-      #haproxy = 97; # DynamicUser as of 2019-11-08
+      haproxy = 97;
```

Comment:
We probably don't need to allocate a fixed uid/gid though. All files could be chowned at startup.

Reply:
AH yeah indeed, see also https://github.com/NixOS/rfcs/blob/master/rfcs/0052-dynamic-ids.md

Reply:
Updated to use dynamically allocated ids instead.